lib/iris/tests/test_pp_cf.py

```diff
@@ -88,7 +88,12 @@ def _test_file(self, name):
             fname_name = name
 
         self.assertCML(
-            cubes, self._ref_dir + ("from_pp", fname_name + ".cml",)
+            cubes,
+            self._ref_dir
+            + (
+                "from_pp",
+                fname_name + ".cml",
+            ),
```

Comment:
This formatting looks really horrible, is less readable and seems gratuitous. Is there a configuration we can use to make Black less zealous?